packages/liquid-html-parser/src/stage-1-cst.spec.ts

```diff
@@ -1011,30 +1011,352 @@ describe('Unit: Stage 1 (CST)', () => {
           expectPath(cst, '0.children.0.value').to.equal('@param');
         });
 
-        it('should parse @param with name', () => {
-          const testStr = `{% doc %} @param paramWithNoDescription {% enddoc %}`;
+        it('should parse required @param', () => {
```

Comment:
In these tests, I typically test permutations, and include `location` assertions for the first permutation only (aside from the whitespace case).

To avoid being overly verbose / repetitive, I'm just checking the value of the `description` / `type` to ensure that nothing has completely broken in the matching logic.